modules/customize-certificates-api-add-named.adoc

```diff
@@ -18,6 +18,17 @@ client's URL.
 reach the API server.
 * The certificate must have the `subjectAltName` extension for the URL.
 
+If a certificate chain is used, all certificates must be manually concatenated into a single named certificate file. This can be achieved with the following example: cat first_cert.pem second_cert.pem third_cert.pem >combined_cert.pem 
+
+The order of the chain should be:
+
+* OpenShift Container Platform master host certificate.
```

Comment:
"OpenShift Container Platform" here still needs to be replaced with "{product-title}".